diff --git a/MODULE.bazel b/MODULE.bazel
index 7e898c6..48a9d45 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -5,7 +5,7 @@ module(
     bazel_compatibility = [">=8.0.0"],
 )

-bazel_dep(name = "abseil-cpp", version = "20260107.1")
+bazel_dep(name = "abseil-cpp", version = "20260817.0")
 bazel_dep(name = "protobuf", version = "35.0")
 bazel_dep(name = "tcmalloc", version = "0.0.0-20250927-12f2552")
 bazel_dep(name = "platforms", version = "1.0.0")
