diff --git a/MODULE.bazel b/MODULE.bazel
index 2c0da7a..4dab588 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -21,8 +21,8 @@ bazel_dep(name = "rules_proto", version = "7.1.0")
 #ifndef PROTO2_OPENSOURCE
 # LINT.IfChange
 #endif // PROTO2_OPENSOURCE
 # protoc dependencies
-bazel_dep(name = "abseil-cpp", version = "20250512.1")
+bazel_dep(name = "abseil-cpp", version = "20260526.0")
 bazel_dep(name = "rules_cc", version = "0.2.18")
 bazel_dep(name = "zlib", version = "1.3.1.bcr.5")
 #ifndef PROTO2_OPENSOURCE
