diff --git a/MODULE.bazel b/MODULE.bazel
index fde6f48..a3e2544 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -19,7 +19,7 @@ module(
 )
 
 bazel_dep(name = "bazel_features", version = "1.20.0")
-bazel_dep(name = "abseil-cpp", version = "20240116.1", repo_name = "com_google_absl")
+bazel_dep(name = "abseil-cpp", version = "20260526.0", repo_name = "com_google_absl")
 bazel_dep(name = "bazel_skylib", version = "1.5.0")
 bazel_dep(name = "platforms", version = "0.0.8")
 bazel_dep(name = "rules_python", version = "1.8.0")
