diff --git a/MODULE.bazel b/MODULE.bazel
index 26074df..c339e38 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -14,7 +14,7 @@ bazel_dep(name = "zlib", version = "1.3.1.bcr.5")
 
 # Required by @remote_java_tools, which is loaded via module extension.
 bazel_dep(name = "rules_license", version = "0.0.3")
-bazel_dep(name = "abseil-cpp", version = "20250814.0", repo_name = "com_google_absl")
+bazel_dep(name = "abseil-cpp", version = "20260526.0", repo_name = "com_google_absl")
 
 # TODO: Remove this override after the next release of abseil-cpp.
 single_version_override(
